src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -513,11 +525,23 @@ START_SECTION(PeakArea integratePeak(
   TEST_REAL_SIMILAR(pa.height, 966489.0)
   TEST_REAL_SIMILAR(pa.apex_pos, 2.7045)
   it = chromatogram.RTBegin(left);
-  for (Size i = 0; i < pa.hull_points.size(); ++i, ++it)
+  for (Size i = 0; i < pa.hull_points.size(); i += 4, it +=4)
   {
     TEST_REAL_SIMILAR(pa.hull_points[i][0], it->getRT())
     TEST_REAL_SIMILAR(pa.hull_points[i][1], it->getIntensity())
   }
+
+  STATUS("Integration type: simpson (1 point)")
+  pa = ptr->integratePeak(chromatogram, left, 2.478);
+  TEST_REAL_SIMILAR(pa.area, 881)
+  TEST_REAL_SIMILAR(pa.height, 881)
+  TEST_REAL_SIMILAR(pa.apex_pos, 2.477966667)
+
+  STATUS("Integration type: simpson (2 points)")
```

Comment:
Let's add a test (once implemented) to asl check for the case of "Integration type: trapezoid (1 point)"